cmd/mobile-server/main.go

```diff
@@ -101,10 +102,13 @@ func main() {
 
 	handler := web.BuildHTTPHandler(router, mwAccess)
 	http.Handle("/", handler)
-	logger.Info("starting server on port " + *port)
-	if err := http.ListenAndServeTLS(*port, *cert, *key, nil); err != nil {
+	logger.Info("starting server on port "+*port, *key, *cert, "target namespace is ", *namespace)
+	if err := http.ListenAndServe(*port, nil); err != nil {
```

Comment:
@maleck13  maybe we should add an arg for using tls or not.
For local dev tls is needed for the oauth flow.
When running in openshift, we could do edge termination

Reply:
I have added changes to allow us to use a cert generated by oscp now see latest commit